activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Log database query callers
+
+    Add `verbose_query_logs` configuration option to display the caller
+    of database queries in the log to facilitate N+1 query resolution
+    and other debugging. Excludes Ruby and Rails callers but not gems.
+
+    False by default in all environments for existing apps but enabled
+    by default in development for new projects in
+    `new_framework_defaults_5_2.rb`.
```

Comment:
`new_framework_defaults_5_2.rb` is generated by the `app:update` rake task, which means it only exists for apps that have upgraded from an older version. The implementation currently does the opposite of what's described here: disabled for new apps, and enabled automatically (in development) for existing apps.

Reply:
Good catch. Let me fix that.

Reply:
@eugeneius Should be addressed now. 

See: 
- https://github.com/rails/rails/pull/26815/files#diff-1ed2907b3b8f148c2533558a77673ffaR7
- https://github.com/rails/rails/pull/26815/files#diff-669ddf1ccccfa4a2b750c27d2d3d0de3R49